js/dashboard-demo/dashboard-updates.js

```diff
@@ -426,6 +426,14 @@ function renderApp() {
 }
 
 function newEventsDataArrived(event) {
+  let tts = event[0].tts;
+  let sound = event[0].sound;
+  if(sound !== undefined || tts !== undefined){
+    console.log(tts);
```

Comment:
Please remove the console.log statements, we don't want to have debuggin information printed al the time in the console